micro_ros_setup/config/host/generic/client-host-colcon.meta

```diff
@@ -1,5 +1,11 @@
 {
     "names":{
+        "*":{
```

Comment:
Is this wildcard documented in colcon .meta files?

Reply:
I didn't find documentation in colcon but I saw this here https://github.com/micro-ROS/micro-ros-build/blob/dashing/micro_ros_setup/config/agent-colcon.meta